caen.hpp

```diff
@@ -599,6 +605,25 @@ class Digitizer {
     return value;
   }
 
+  /* helper class to translate Acquisition Status of register 0x8104 in X740 and X751 */
+  class AcquisitionStatus {
+  private:
+    uint32_t v;
+
+  public:
+    AcquisitionStatus(uint32_t value) : v(value) {}
```

Comment:
clearly these are bit testing attempts, would be better to replace with ISBITON() macro and possibly using
const defines for each bit assigning each a name
const int status =           0x00000002
const int event_ready = 0x00000004
etc. and maybe with a comment pointing to the relevant page of the documentation

Reply:
Sounds reasonable. I actually just shifted this code around a bit since it was just a copy+paste for each digitizer type.
Are there any specific problems with the current implementation approach then?

Reply:
no it works fine, it is just that the intention is more clear to write 
ISBITON(event_ready)

In general we might want to make a digitizer register map class, so all
magic numbers are located together.

I don't think this should be holding us back at all.